tools/DevDiagnostics/DevHome.DevDiagnostics/Helpers/Tool.cs

```diff
@@ -71,6 +72,10 @@ public void Invoke()
         ToolLaunchOptions options = new();
         options.TargetProcessId = TargetAppData.Instance.TargetProcess?.Id;
         options.TargetHWnd = TargetAppData.Instance.HWnd;
+
+        var barWindow = Application.Current.GetService<PrimaryWindow>().DBarWindow;
+        options.ParentWindow = barWindow;
```

Comment:
Does this mean that if the Bar window is closed, that the child window will be automatically closed too?

Reply:
That happens already; this doesn't make any difference to that behavior.